### PR TITLE
Test: Add active cover amount tracking unit tests

### PR DESCRIPTION
**Pull request by patitonar**
## Context

Relates to #278 

## Changes proposed in this pull request

Adds new unit tests for `enableActiveCoverAmountTracking`, `commitActiveCoverAmounts`, and `expireCover`. Implemented the list of test cases from the issue, except for the following ones that are going to be submitted in another PR as integration tests:
- `Should expire a cover that had a claim paid out fully`
- `Should expire a cover that had partial claim paid out`
- `Should expire a cover that had rejected claim`

## Test plan

New tests were added in `test/unit/Cover/enableActiveCoverAmountTracking.js`, `test/unit/Cover/commitActiveCoverAmounts.js` and `test/unit/Cover/expireCover.js`

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [x] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [x] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: test/unit/Cover/enableActiveCoverAmountTracking.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    await cover.connect(emergencyAdmin).enableActiveCoverAmountTracking([], []);
+    await cover.connect(emergencyAdmin).commitActiveCoverAmounts();
+
+    await expect(cover.connect(emergencyAdmin).enableActiveCoverAmountTracking([], [])).to.be.revertedWith(
+      'Cover: activeCoverAmountCommitted is already true',
+    );
+  });
+
+  it('reverts if active cover amounts already committed, even if cover tracking was never enabled', async function () {
```
Comment (danoctavian) [inline test/unit/Cover/enableActiveCoverAmountTracking.js:108]: good edge case captured here

File: test/unit/Cover/expireCover.js [hunk 2]
```diff
@@ -40,14 +39,44 @@ describe('expireCover', function () {
...
+
+    await time.increase(ethCoverBuyFixture.period + 1);
+
+    const coverId = 0;
+
+    await cover.connect(anyone).expireCover(coverId);
 
     const activeCoverAmountAfterExpiry = await cover.totalActiveCoverInAsset(coverAsset);
-    bnEqual(activeCoverAmountAfterExpiry, parseEther('0'));
+    expect(activeCoverAmountAfterExpiry).to.be.equal(0);
```
Comment (danoctavian) [inline test/unit/Cover/expireCover.js:75]: good cleanup

**Review by danoctavian — APPROVED** [review]
LGTM without other changes

**Comment by danoctavian** [comment]
Rebase here please
